lizmap/modules/lizmap/lib/Request/OGCRequest.php

```diff
@@ -185,6 +185,68 @@ protected function constructUrl()
         return Proxy::constructUrl($this->parameters(), $this->services, $url);
     }
 
+    /**
+     * Generate a string to identify the target of the HTTP request.
+     *
+     * @param array $params The list of HTTP params
+     * @param int   $code   The HTTP code of the request
+     */
+    private function formatHttpErrorString($params, $code)
+    {
+        $message = '';
+
+        if (array_key_exists('map', $params)) {
```

Comment:
may be build message with a loop ?

```suggestion
    $paramsToLog = ['map', 'repository', 'project', 'service', 'request']
    foreach($paramsToLog as $paramName)  {
        if (array_key_exists($paramName, $params)) {
            $message .= strtoupper($paramName)." '".$params[paramName]."', ";
        }
    }
```

Reply:
Indeed, thanks.
@rldhont told me he wants to look at his PR as well.